torch/nn/modules/module.py

```diff
@@ -221,6 +223,11 @@ def register_module_forward_hook(hook: Callable[..., None]) -> RemovableHandle:
     it will not have effect on forward since this is called after
     :func:`forward` is called.
 
+    Parameters:
+        hook (Callable): The user defined hook to be registered.
+        force (bool): If ``True`` the ``hook`` will be run regardless of
```

Comment:
I'm not sure of this name, I would also propose "always_{run,call}", "{run,call}_on_exception" ?